src/features/content-explorer/item-list/ItemList.js

```diff
@@ -188,8 +190,30 @@ const ItemList = ({
         };
     }
 
+    const addAutoSizer = WrappedComponent => {
+        return props => {
+            return isResponsive ? (
+                <div style={{ flex: 1 }}>
+                    <AutoSizer>
+                        {({ width: w, height: h }) => <WrappedComponent {...props} width={w} height={h} />}
```

Comment:
I think it's okay to use the full names here instead `w` and `h`.
```suggestion
                        {({ height, width }) => <WrappedComponent height={height} width={width} {...props} />}
```

Reply:
Using standard `width` and `height` actually gave me linting errors:
<img width="1143" alt="Screen Shot 2022-07-07 at 3 52 37 PM" src="https://user-images.githubusercontent.com/6400298/177884531-00640822-535a-4bc7-b0d9-a4c8fb70cfb0.png">